sdk/identity/azure-identity/pom.xml

```diff
@@ -93,6 +93,18 @@
       <version>2.8.5</version> <!-- {x-version-update;com.google.code.gson:gson;external_dependency} -->
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>msal4j-persistence-extension</artifactId>
+      <version>0.1</version> <!-- {x-version-update;com.microsoft.azure:msal4j-persistence-extension;external_dependency} -->
+    </dependency>
+
+    <dependency>
+      <groupId>org.linguafranca.pwdb</groupId>
+      <artifactId>KeePassJava2</artifactId>
+      <version>2.1.4</version> <!-- {x-version-update;org.linguafranca.pwdb:KeePassJava2;external_dependency} -->
+    </dependency>
```

Comment:
Are these approved external dependencies?

@JonathanGiles

Reply:
I am ok with them, as long as @g2vinay has followed the requirements of the license.

Reply:
as per @anuchandy , if I am using maven dependency, then we don't need to do anything else.